app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
+  has_many :likes
+  has_many :comments
+
```

Comment:
- [ ] To correctly match the associations between models  according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) and as required for the project, kindly ensure that the `Post` belongs to an `:author`, so after you can add the reference of the class. It should look to some thing like this:
```ruby
belongs_to :author, class_name: 'User'
has_many :comments
has_many :likes
```

Reply:
@tresorsawasawa There seems to be a discrepancy in the diagrams. I don't think this change is necessary
- Diagram from the project - [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png)
- Diagram you attached in your review - [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)

Reply:
Hi @newhoteng 

I really apologizing to you. After the call we did, I noticed that the link of the[ project's requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project_v1_1.md) given to me it seems to have been a discrepancy. I think